From: Paul Gortmaker <paul.gortmaker@windriver.com>
Date: Wed, 8 Jan 2014 15:32:47 -0500
Subject: drivers/net: delete non-required instances of include <linux/init.h>
Patch-mainline: v3.14-rc1
Git-commit: a81ab36bf52d0ca3a32251a923be1dbced726141 (partial)
References: FATE#315952

None of these files are actually using any __init type directives
and hence don't need to include <linux/init.h>.   Most are just a
left over from __devinit and __cpuinit removal, or simply due to
code getting copied from one driver to the next.

This covers everything under drivers/net except for wireless, which
has been submitted separately.

Signed-off-by: Paul Gortmaker <paul.gortmaker@windriver.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: Benjamin Poirier <bpoirier@suse.de>
---
 drivers/net/ethernet/chelsio/cxgb3/common.h |    1 -
 1 file changed, 1 deletion(-)
--- a/drivers/net/ethernet/chelsio/cxgb3/common.h
+++ b/drivers/net/ethernet/chelsio/cxgb3/common.h
@@ -36,7 +36,6 @@
 #include <linux/types.h>
 #include <linux/ctype.h>
 #include <linux/delay.h>
-#include <linux/init.h>
 #include <linux/netdevice.h>
 #include <linux/ethtool.h>
 #include <linux/mdio.h>
